### PR TITLE
news page 1 added

### PR DESCRIPTION
**Pull request by Iyiola-stack**
I added News page 1

### CODE REVIEW

File: peterIyiola/news-1.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (goke-epapa) [inline peterIyiola/news-1.html]: Please remove the extra empty lines
```suggestion
```

File: peterIyiola/news-1.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>Coronavirus:How to protect yourself</title>
+   </head>
+<body>
```
Comment (goke-epapa) [inline peterIyiola/news-1.html]: Please indent the `<body>` tag and it's content

File: peterIyiola/news-1.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <p><b>6. Inform the health authorities</b></p>
+        <p>Anyone who shows symptoms and has been in one of the severely affected areas or has had contact with people who have stayed there should contact the health authorities at an early stage. For this reason, the <a href "https://ncdc.gov.ng/">NCDC</a> has established a coronavirus hotline. It is better not to simply go to the doctor or hospital, as you could infect others on the way there and in the hospital itself. The hotline can be reached under (0800 9700 0010) </p>
+        <p><b>7. Get tested for coronavirus</b></p>
+        <p>Anyone who has symptoms, has had contact with a confirmed case or has been in a risk area himself can be tested for the coronavirus. The costs are only covered for suspected cases if the above-mentioned conditions are met. </p>
+
```
Comment (goke-epapa) [inline peterIyiola/news-1.html]: Please remove unnecessary blank lines

**Review by goke-epapa — COMMENTED** [review]
Good, please check the comments

**Review by goke-epapa — APPROVED** [review]

**Review by faithakinbobola — APPROVED** [review]
Nicely indented now

**Pull request merged**